packages/oscal-react-library/src/components/OSCALCatalogBaseline.tsx

```diff
@@ -520,6 +478,472 @@ export default function OSCALCatalogBaseline() {
+      setStartDropping(false);
+      setEndUploading(true);
+    }
+    function handleGoBack() {
+      setStartDropping(false);
+      setEndUploading(false);
+      setUploadSuccessfulStatus(false);
+    }
+    function handleOpenFile() {
+      const request_json = {
+        oscal_file: newOSCALFilePath,
+      };
+      setUploadSuccessfulStatus(true);
+      setUpload(true);
+      setUploadNewCatalogBaseline(false);
+      fetchTransaction("/get_metadata", request_json, getMetadataSuccess, getMetadataFail);
+      function getMetadataSuccess(response: any) {
+        console.log("newly response", response);
+        const model: CatalogBaseline = {
+          title: response.title,
+          lastModified: response.lastModified,
+          documentVersion: response.version,
+          publicationDate: response.publicationDate,
+          projectUUID: response.projectUUID,
+          isVisible: true,
+        };
+        setNewOSCALModel(model);
+        setOpenCatalogBaseline(true);
+      }
+      function getMetadataFail(e: any) {
+        console.log("In FilledBoxItem: Operation fail ", e.statusText);
+      }
+    }
+    return (
+      <>
+        {!upload && !uploadSuccessful && (
+          <Box sx={{ width: 300, height: 400 }} overflow={"hidden"}>
+            <MainImage
+              sx={{ top: 330, left: 470, width: 150, position: "absolute" }}
+              src={FileIcon}
+            />
+            <UploadTypo
+              variant="h1"
+              sx={{
+                top: 510,
+                left: 400,
+                width: 400,
+                height: 30,
+                position: "absolute",
+              }}
+            >
+              No catalogs or baselines defined!
+            </UploadTypo>
+            <OSCALPrimaryButton
+              sx={{ top: 550, left: 450, width: 91, height: 36, position: "absolute" }}
+              onClick={handleUpload}
+            >
+              <Typography>UPLOAD</Typography>
+            </OSCALPrimaryButton>
+            <OSCALTertiaryButton
+              sx={{ top: 550, left: 540, width: 140, height: 36, position: "absolute" }}
+              onClick={handleAddNewCatalogBaseline}
+            >
+              <Typography
+                sx={{
+                  fontFamily: "Source Sans Pro",
+                  fontSize: 16,
+                  fontWeight: 600,
+                  lineHeight: 20,
+                  letterSpacing: 0,
+                  textAlign: "left",
+                  color: "#023E88",
+                }}
+              >
+                CREATE NEW +
+              </Typography>
+            </OSCALTertiaryButton>
+          </Box>
+        )}
+        {upload && (
+          <Box
+            sx={{
+              top: 180,
+              left: 40,
+              height: 400,
+              width: 1030,
+              background: "#ffffff",
+              position: "absolute",
+              border: "1xp solid #0028675E",
```

Comment:
Just wanted to just mention it once more, in the future we should try to keep the units of measure to `rem`, where we are sizing/positioning components.